File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.java
import java.io.*;
import java.util.*;

public class Main{
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int N = Integer.parseInt(st.nextToken());
        int[] nums = new int[N];
        for(int i=0; i<N; i++) {
            st = new StringTokenizer(br.readLine());
            nums[i] = Integer.parseInt(st.nextToken());
        }

        for(int i=0; i<N-1; i++) {
            for(int j=i+1; j<N; j++) {
                if(nums[i] > nums[j]) {
                    int temp = nums[i];
                    nums[i] = nums[j];
                    nums[j] = temp;
                }
            }
        }

        for(int i=0; i<N; i++) {
            bw.write(String.valueOf(nums[i]));
            bw.newLine();
        }
        bw.flush();
    }

}

File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.cc
#include <iostream>

using namespace std;

int main() {
    int N;
    cin >> N;

    int nums[N];

    for(int i=0; i<N; i++) {
        cin >> nums[i];
    }

    for(int i=N-1; i>=0; i--) {
        for(int j=0; j<i; j++) {
            if(nums[j] > nums[j+1]) {
                int temp = nums[j];
                nums[j] = nums[j+1];
                nums[j+1] = temp;
            }
        }
    }

    for(int i=0; i<N; i++) {
        cout << nums[i] << endl;
    }

    return 0;
}